README.md

```diff
@@ -546,6 +546,7 @@ issue](https://github.com/brigade/overcommit/issues/238) for more details.
 * [SlimLint](lib/overcommit/hook/pre_commit/slim_lint.rb)
 * [Sqlint](lib/overcommit/hook/pre_commit/sqlint.rb)
 * [Standard](lib/overcommit/hook/pre_commit/standard.rb)
+* [StyleLint](lib/overcommit/hook/pre_commit/style_lint.rb)
```

Comment:
Nit: If the actual project name is capitalized "Stylelint", then we should use that same capitalization, and have the file be named `stylelint.rb` appropriately.